src/main/java/com/example/decafe/Customer.java

```diff
@@ -81,17 +77,8 @@ public ImageView getLabel() { //returns the label of the customer
         return this.orderLabel;
     }
 
-    public String getRandomOrder() { //returns random order
-
-        Random random = new Random();
-        int number = random.nextInt(2);
-
-        switch (number) {
-            case 0 -> order = "cake";
-            case 1 -> order = "coffee";
-        }
-
-        return order;
+    public String getRandomOrder() {
```

Comment:
Can you please use here the CafeAssets for the strings : CafeAssets.CAKE.getName() and CafeAssets.COFFEE.getName()